src/frontend/Komponenter/Behandling/Endring/EndringPersonopplysninger.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { RessursStatus } from '../../../App/typer/ressurs';
+import { useHentVilkår } from '../../../App/hooks/useHentVilkår';
+import { Alert, Heading } from '@navikt/ds-react';
+import DataViewer from '../../../Felles/DataViewer/DataViewer';
+import { utledEndringerPåPersonopplysninger } from './utils';
+import { useToggles } from '../../../App/context/TogglesContext';
+import { ToggleName } from '../../../App/context/toggles';
+import { endringerKeyTilTekst } from '../Inngangsvilkår/vilkår';
+
+const EndringPersonopplsyninger: React.FC<{
+    behandlingId: string;
+}> = ({ behandlingId }) => {
+    const { endringerPersonopplysninger, hentEndringerForPersonopplysninger } = useHentVilkår();
+    const { toggles } = useToggles();
+    const skalViseKomponent = toggles[ToggleName.visEndringerPersonopplysninger];
+
+    React.useEffect(() => {
+        if (behandlingId !== undefined) {
+            if (endringerPersonopplysninger.status !== RessursStatus.SUKSESS) {
+                hentEndringerForPersonopplysninger(behandlingId);
+            }
+        }
+        // eslint-disable-next-line
+    }, [behandlingId]);
+
+    if (!skalViseKomponent) {
+        return <></>;
+    }
+
+    return (
+        <DataViewer response={{ endringerPersonopplysninger }}>
+            {({ endringerPersonopplysninger }) => {
+                const endringer = utledEndringerPåPersonopplysninger(endringerPersonopplysninger);
+                const erEndring = endringer.length > 0;
+
+                if (!erEndring) {
+                    return <></>;
+                }
+
+                return (
+                    <Alert variant="warning" title="Endring i folkeregisteropplysninger">
+                        <Heading spacing size="small" level="2">
+                            Endring i folkeregisteropplysninger
```

Comment:
Burde vi opplyse om at de bør trykke på "oppdater registeropplysninger" for å få oppdatert registerdata i løsningen?

Reply:
Bedre tekst kommer senere :-)